deployment/k8s/charts/components/bootstrapK8s/templates/role.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.serviceAccountName }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/component: bootstrap-k8s
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: migration-role
```

Comment:
Should this be more targeted to the bootstrapping phase or is it used afterwards for migrations?